Sources/GroundState/py_variational_montecarlo.hpp

```diff
@@ -160,15 +160,198 @@ void AddVariationalMonteCarloModule(py::module &m) {
       .def(
           "get_observable_stats",
           [](VariationalMonteCarlo &self) {
-            py::dict data;
             self.ComputeObservables();
-            self.GetObsManager().InsertAllStats(data);
-            return data;
+            return self.GetObservableStats();
           },
           R"EOF(
         Calculate and return the value of the operators stored as observables.
 
-        )EOF");
+        )EOF")
+      .def_property_readonly("vmc_data", &VariationalMonteCarlo::GetVmcData);
+
+  py::class_<vmc::Result>(m_vmc, "_VmcResult");
+
+  m_vmc.def("compute_samples", vmc::ComputeSamples, py::arg("sampler"),
```

Comment:
is this `compute_samples` or `compute_vmc_samples` as in the docs?

Reply:
I renamed it, but forgot to change it in the docs.